src/openforms/js/compiled-lang/nl.json

```diff
@@ -35,10 +49,28 @@
       "value": "Naam"
     }
   ],
+  "5P+YgL": [
+    {
+      "type": 0,
+      "value": "Stap toevoegen"
+    }
+  ],
+  "5zieWh": [
+    {
+      "type": 0,
+      "value": "Naam van de formulierdefinitie die in deze stap gebruikt wordt."
+    }
+  ],
+  "6IAwkJ": [
+    {
+      "type": 0,
+      "value": "Inloggen is vereist voor deze stap, maar er is geen inlogoptie geselecteerd op het formulier."
+    }
+  ],
   "6PIG4q": [
     {
       "type": 0,
-      "value": "Inlogopties"
+      "value": "Authentication"
```

Comment:
Is it a conscious choice to leave this in english in the corresponding nl.json?

Reply:
nope! must be something wrong in the translation tooling :grimacing: